apiserver/facades/agent/caasoperator/mock_test.go

```diff
@@ -100,15 +105,17 @@ type mockApplication struct {
 	forceUpgrade bool
 	unitsChanges chan []string
 	unitsWatcher *statetesting.MockStringsWatcher
+	appChanges   chan struct{}
+	watcher      *statetesting.MockNotifyWatcher
 }
 
 func (*mockApplication) Tag() names.Tag {
-	panic("should not be called")
+	return names.NewApplicationTag("gitlab")
 }
 
-func (a *mockApplication) Life() state.Life {
-	a.MethodCall(a, "Life")
-	return a.life
+func (app *mockApplication) Life() state.Life {
```

Comment:
Not sure why you did not like just an 'a' (pointers generally are a one letter) but, sure, 'app' works too :D

Reply:
renamed